### PR TITLE
[REVIEW] Update Merlin Inference API TF PoC notebooks 

### PR DESCRIPTION
**Pull request by rnyak**
The notebooks and readme are being updated wrt the bug bash comments.

### CODE REVIEW

File: examples/inference_triton/README.md [hunk 1]
```diff
@@ -116,6 +132,6 @@ I0216 15:28:24.027067 71 http_server.cc:2717] Started HTTPService at 0.0.0.0:800
 I0216 15:28:24.068597 71 http_server.cc:2736] Started Metrics Service at 0.0.0.0:8002
 ```
 
-All the models should show "READY" status to indicate that they loaded correctly. If a model fails to load the status will report the failure and a reason for the failure. If your model is not displayed in the table check the path to the model repository and your CUDA drivers.
+All the models should show "READY" status to indicate that they are loaded correctly. If a model fails to load the status will report the failure and a reason for the failure. If your model is not displayed in the table, check the path to the model repository and your CUDA drivers.
```
Comment (benfred) [inline examples/inference_triton/README.md:135]: By default triton won't start if the models don't load - maybe we should add a note here?

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

File: examples/inference_triton/README.md (hunk 1 above)
Comment (rnyak) [reply examples/inference_triton/README.md:135]: > By default triton won't start if the models don't load - maybe we should add a note here?

Ok will add it.

**Review by benfred — APPROVED** [review]

**Pull request merged**